code/modules/modular_computers/file_system/programs/jobmanagement.dm

```diff
@@ -14,13 +14,13 @@ GLOBAL_VAR_INIT(time_last_changed_position, 0)
 	program_icon = "address-book"
 
 	var/change_position_cooldown = 30
-	//Jobs you cannot open new positions for
+	///Jobs blacklisted from having their slots edited.
 	var/list/blacklisted = list(
 		JOB_CAPTAIN,
 		JOB_HEAD_OF_PERSONNEL,
 		JOB_HEAD_OF_SECURITY,
 		JOB_RESEARCH_DIRECTOR,
-		JOB_CHIEF_MEDICAL_OFFICER,
```

Comment:
oops

Reply:
it was actually my fault (which was why I put GBP no update), I caused it in https://github.com/tgstation/tgstation/pull/63357/files